### ISSUE TITLE
Registration Error

### ISSUE THREAD

**Issue by sidf3ar** [zelon88/Resource_Monitor#1]
I tried to sign myself for the cloud services on your website but the registration was unsuccessful. Are you still running the cloud??

**Comment by zelon88** [comment]
Thanks for the heads up! We are still running the Cloud.

I just checked out the registration process by creating an account myself. I entered a new username and an unused email account. I also made sure to check off the "Consent" checkbox. 

The account was created successfully and the signup email was received. I was able to change my password for the account and log into it. I also verified that HRCloud2 does work on a fresh account.

This might have to do with the country you're making requests from. We've had to geofence certain locations due to excessive spam and attack. Currently we do not offer accounts to Belarus, China, Egypt, Hong Kong, Russia, Ukraine, Pakistan or Vietnam. Additionally we do not process HTTP or HTTPS requests from Iran. Nothing political, it's just not worth serving these regions as >99% of requests from these countries are malicious.

If possible try using a VPN or Tor and see if that corrects the problem. If those are not an option for you I can provide you with a bypass code that allows you to use the site from within geofenced regions.
